open_city_profile/tests/snapshots/snap_test_graphql_api_schema.py

```diff
@@ -182,18 +182,25 @@
 }
 
 type DeleteMyProfileMutationPayload {
+  results: [DeletionResultNode]
```

Comment:
Array probably won't contain nulls: `results: [DeletionResultNode!]`.

Is there always an array or can the array itself be null? Could this be even `results: [DeletionResultNode!]!`?